File: api/Models/ClassRoom.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;
namespace api.Models
{
    [Table("Ucionica")]
    public class ClassRoom
    {
        [Column("id_ucionice")]
        public int Id { get; set; }
        [Column("broj_ucionice")]
        public int classRoomNumber { get; set; }
        [Column("kapacitet")]
        public int Capacity { get; set; }
    }
}

File: api/app/Models/ClassRoom.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
class ClassRoom extends Model
{
    use HasFactory;
    const TABLE = 'classrooms';
    protected $table = self::TABLE;
    protected $fillable = [
        'classroom_number',
        'capacity',
    ];
}